From 560404ad7b5ed16198d5e7fb0e7b983fdd8eb2eb Mon Sep 17 00:00:00 2001
From: Xu Jianqun <jay.xu@rock-chips.com>
Date: Tue, 5 Jan 2016 19:30:31 +0800
Subject: [PATCH] ARM64: dts: rockchip: move i2c0 node more ahead

Move i2c0 node more forhead than other i2c nodes,
to make sure that the pmic on i2c0 bus will be
probed firstly, which supplys for some devices.

Change-Id: Ibbc43307cdc2701eb73e157c9e73e9b7f1af8b3b
Signed-off-by: Xu Jianqun <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3368.dtsi | 26 ++++++++++++------------
 1 file changed, 13 insertions(+), 13 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368.dtsi b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
index 0134ef33504f..44469bc064d3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3368.dtsi
@@ -333,6 +333,19 @@
 		status = "disabled";
 	};
 
+	i2c0: i2c@ff650000 {
+		compatible = "rockchip,rk3368-i2c", "rockchip,rk3288-i2c";
+		reg = <0x0 0xff650000 0x0 0x1000>;
+		clocks = <&cru PCLK_I2C0>;
+		clock-names = "i2c";
+		interrupts = <GIC_SPI 60 IRQ_TYPE_LEVEL_HIGH>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&i2c0_xfer>;
+		#address-cells = <1>;
+		#size-cells = <0>;
+		status = "disabled";
+	};
+
 	i2c2: i2c@ff140000 {
 		compatible = "rockchip,rk3368-i2c", "rockchip,rk3288-i2c";
 		reg = <0x0 0xff140000 0x0 0x1000>;
@@ -479,19 +492,6 @@
 		reg = <0x0 0xff610000 0x0 0x400>;
 	};
 
-	i2c0: i2c@ff650000 {
-		compatible = "rockchip,rk3368-i2c", "rockchip,rk3288-i2c";
-		reg = <0x0 0xff650000 0x0 0x1000>;
-		clocks = <&cru PCLK_I2C0>;
-		clock-names = "i2c";
-		interrupts = <GIC_SPI 60 IRQ_TYPE_LEVEL_HIGH>;
-		pinctrl-names = "default";
-		pinctrl-0 = <&i2c0_xfer>;
-		#address-cells = <1>;
-		#size-cells = <0>;
-		status = "disabled";
-	};
-
 	i2c1: i2c@ff660000 {
 		compatible = "rockchip,rk3368-i2c", "rockchip,rk3288-i2c";
 		reg = <0x0 0xff660000 0x0 0x1000>;
-- 
2.35.3

